pyproject.toml

```diff
@@ -65,8 +65,9 @@ dependencies = [
     # error reports
     "sentry-sdk",
 
-    # Database Video html Site Generator 
+    # Database Video html Site Generator
     "htmlmin",
+    "ruamel.yaml",
```

Comment:
I wonder if these two could be an extra which we don't include in the executable.

Reply:
certainly could be, although out of scope for this pr imo

Reply:
I'd prefer that done before the next release though.

Reply:
kk. you'll have to show mii how to do it